health/micro-ui/web/micro-ui-internals/packages/modules/microplan/README.md

```
@@ -0,0 +1,96 @@
+- [abishek-egov](https://github.com/abhishek-egov)
+- [ramkrishna-egov](https://github.com/ramkrishna-egov)
+- [ashish-egov](https://github.com/ashish-egov)
+- [rachna-egov](https://github.com/rachna-egov)
+
+## License
+
+[MIT](https://choosealicense.com/licenses/mit/)
+
+## Documentation
+
+Documentation Site (https://core.digit.org/guides/developer-guide/ui-developer-guide/digit-ui)
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)